public/fonts/fonts.css

```diff
@@ -1,54 +1,54 @@
 @font-face {
     font-family: "ISO";
-    src: url("ISOv0.9-Thin.woff2") format('woff2');
+    src: url("VCTRMonov0.12-Thin.woff2") format('woff2');
     font-style: normal;
     font-weight: 100;
   }
   
   @font-face {
     font-family: "ISO";
-    src: url("ISOv0.9-Light.woff2") format('woff2');
+    src: url("VCTRMonov0.12-Light.woff2") format('woff2');
     font-style: normal;
     font-weight: 200;
   }
   
   @font-face {
       font-family: "ISO";
-      src: url("ISOv0.9-Regular.woff2") format('woff2');
+      src: url("VCTRMonov0.12-Regular.woff2") format('woff2');
       font-style: normal;
       font-weight: 400;
     }
 
   @font-face {
     font-family: "ISO";
-    src: url("ISOv0.9-RegularItalic.woff2") format('woff2');
+    src: url("VCTRMonov0.12-RegularItalic.woff2") format('woff2');
     font-style: italic;
     font-weight: 400;
     }
   
   @font-face {
     font-family: "ISO";
-    src: url("ISOv0.9-Medium.woff2") format('woff2');
+    src: url("VCTRMonov0.12-Medium.woff2") format('woff2');
     font-style: normal;
     font-weight: 600;
   }
   @font-face {
     font-family: "ISO";
-    src: url("ISOv0.9-Medium.woff2") format('woff2');
+    src: url("VCTRMonov0.12-MediumItalic.woff2") format('woff2');
     font-style: normal;
```

Comment:
Can you change `font-style: normal` to `font-style: italic`?  (since we already have font-style: normal for 600 above)

Reply:
sure